javascript/exorelay/features/support/steps-given.ls

```diff
@@ -19,6 +20,12 @@ module.exports = ->
     @exo-relay = new ExoRelay {exocom-host: 'localhost', @exocom-port, role: @role = 'test-service'}
 
 
+  @Given /^an ExoRelay instance is connected at port (\d+)$/ (@exocom-port, done) ->
+    @exo-relay = new ExoRelay {exocom-host: 'localhost', @exocom-port, role: @role = 'test-service'}
+      ..connect!
+      ..on 'online', ~> done!
```

Comment:
I've never seen event emitters follow `(err, result)` as they typically have a dedicated `error` event. So this is fine